dlt-connector/src/graphql/resolver/CommunityResolver.ts

```diff
@@ -0,0 +1,78 @@
+import { Resolver, Query, Arg, Mutation } from 'type-graphql'
+
+import { CommunityDraft } from '@input/CommunityDraft'
+
+import { getDataSource } from '@typeorm/DataSource'
+
+import { TransactionResult } from '../model/TransactionResult'
+import { TransactionError } from '../model/TransactionError'
+import {
+  create as createCommunity,
+  iotaTopicFromCommunityUUID,
+  isExist,
+} from '@/controller/Community'
+import { TransactionErrorType } from '../enum/TransactionErrorType'
+import { logger } from '@/server/logger'
+
+@Resolver()
+export class CommunityResolver {
+  // Why a dummy function?
+  // to prevent this error by start:
+  //   GeneratingSchemaError: Some errors occurred while generating GraphQL schema:
+  //     Type Query must define one or more fields.
+  // it seems that at least one query must be defined
+  // https://github.com/ardatan/graphql-tools/issues/764
+  @Query(() => String)
+  version(): string {
+    return '0.1'
+  }
+
+  @Mutation(() => TransactionResult)
+  async addCommunity(
+    @Arg('data')
+    communityDraft: CommunityDraft,
+  ): Promise<TransactionResult> {
+    try {
+      const topic = iotaTopicFromCommunityUUID(communityDraft.uuid)
+      // check if community was already written to db
+      if (await isExist(topic)) {
+        return new TransactionResult(
+          new TransactionError(TransactionErrorType.ALREADY_EXIST, 'community already exist!'),
+        )
+      }
+      const community = createCommunity(communityDraft, topic)
+
+      let result: TransactionResult
+
+      if (!communityDraft.foreign) {
+        // TODO: CommunityRoot Transaction for blockchain
+      }
+
+      const queryRunner = getDataSource().createQueryRunner()
+      await queryRunner.connect()
+      await queryRunner.startTransaction()
+
+      try {
+        await queryRunner.manager.save(community)
+        await queryRunner.commitTransaction()
+        result = new TransactionResult()
```

Comment:
Because in case of home community in future there will be more than one transaction which should be saved together. 
But you right, in this state I don't need it and it is maybe better to use query runner and transaction only if I really need them, even if the code is than longer.